project/src/main/java/com/google/sps/servlets/NewsService.java

```diff
@@ -15,35 +15,59 @@
 package com.google.sps.servlets;
 
 import okhttp3.*;
+import com.rometools.rome.feed.synd.SyndFeed;
+import com.rometools.rome.feed.synd.SyndEntry;
+import com.rometools.rome.io.SyndFeedInput;
+import com.rometools.rome.io.XmlReader;
 
-import java.io.IOException;
+import java.net.*;
+import java.util.*;
 
 /** Service class that delivers news under a requested topic. */
 public class NewsService {
   // one instance, reuse
   private final OkHttpClient httpClient = new OkHttpClient();
 
   // Retrieves articles from the Google News RSS Feed using the topic parameter as the search query
-  // The Topic parameter allows for spaces and non alphanumeric characters, but null is an invalid argument
-  public String retrieveNewArticles(String topic) {
+  // The Topic parameter allows for spaces and non alphanumeric characters, but null and "" is an invalid argument
```

Comment:
nit : ... non-alphanumeric character. Null and "" are invalid arguments

Reply:
super nit: are instead of is  :)